content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -125,7 +125,8 @@ this occurs, the applier has 3 options to resolve the conflicts:
 
 * **Overwrite value, become sole manager:** If overwriting the value was
   intentional (or if the applier is an automated process like a controller) the
-  applier should set the `force` query parameter to true and make the request
+  applier should add `--force-conflicts` to `kubectl apply` or set
+  the `force` query parameter to true and make the request
```

Comment:
Thanks for the suggested change, this is still an API-driven doc so I would phrase it with that in mind first (but agree it's convenient to say how it translates into kubectl).

So maybe:
```suggestion
  applier should set the `force` query parameter to true (in kubectl, it can be done by
  using the `--force-conflicts` flag with the apply command) and make the request
```

WDYT?

Reply:
Ah I see. Yea looks great! Thanks